### PR TITLE
Release/0.4.0

### PR DESCRIPTION
**Pull request by chuwy**

### CODE REVIEW

**Comment by chuwy** [comment]
This release should be blocked by https://github.com/snowplow/iglu-central/pull/304.
* [Criterion](https://github.com/snowplow/iglu-scala-client/blob/release/0.4.0/src/main/scala/com.snowplowanalytics.iglu/client/Resolver.scala#L54)
* [Schema](https://github.com/snowplow/iglu-scala-client/blob/release/0.4.0/src/main/resources/iglu-client-embedded/schemas/com.snowplowanalytics.iglu/resolver-config/jsonschema/1-0-1)

**Comment by chuwy** [comment]
Just in case @alexanderdean, I'm receiving cryptic `NoSuchMethodError` in Hadoop Enrich. It probably would be better to not release it until I handle it, to be sure it is not caused by Iglu Client 0.4.0.

**Comment by alexanderdean** [comment]
Okay @chuwy - thanks for the heads-up!

**Comment by chuwy** [comment]
All tests in enrichment are passed successfully. This is ready for review and release.

**Comment by alexanderdean** [comment]
Cool! @fblundun can you take a look at this one?

File: README.md [hunk 1]
```diff
@@ -6,6 +8,48 @@ Iglu Scala Client is used extensively in **[Snowplow] [snowplow-repo]** to valid
...
 
+## Installation
+
+The latest version of Iglu Scala Client is 0.4.0, which is cross-built against Scala 2.10.x and 2.11.x.
+
+If you're using SBT, add the following lines to your build file:
+
+```scala
+// Resolvers
+val snowplowRepo = "SnowPlow Repo" at "http://maven.snplow.com/releases/"
```
Comment (fblundun) [inline README.md]: "SnowPlow" should be "Snowplow".

**Comment by fblundun** [comment]
@chuwy this PR looks good to me!

**Comment by alexanderdean** [comment]
Hey @chuwy - this is looking really good. Nice work. I'd like to spend 20 mins talking through the code - but before we do this, I wanted to check: have you:

* Published local as an -M1
* Updated Scala Common Enrich to use it, then tested that
* Updated Scala Hadoop Enrich to use the updated Scala Common Enrich, then tested that
* Updated Scala Hadoop Shred to use the updated Iglu Scala Client, then tested that

Basically I want to know that there are no regressions before we release this...

**Comment by chuwy** [comment]
Yes, I tested everything on Thursday. It compiles, running and seems to work correct. There was a regression at first, but I fixed and pushed it.

**Comment by alexanderdean** [comment]
This is all released now!

**Comment by chuwy** [comment]
Nice.
